### DISCUSSION TITLE
Use or build from source ddev that runs natively on linux without requiring the docker container

### DISCUSSION THREAD

**Discussion by bbholty1** [ddev/ddev#6765] [Q&A] [answered]
Is this possible??

The issue i'm having is the native docker container I already use on my Linux PC Isn't compatible with the non-native docker container required by DDEV.

Thanks in Advance!
-Bryan

**Comment by rfay** [comment]
Hi @bbholty1 - DDEV is a docker application. 

If you explain what the actual problem is that you're having, we're happy to help. DDEV uses its own docker images. If you're using the normal Docker installation on your Linux machine, it should work fine.

**Reply by bbholty1** [reply]
Thanks for the reply.. i'll give it another shot. I'm pretty sure I got an error message from ddev regarding some sort of container error with the native docker container installed on my system...  If it happens again i'll let you know.

**Reply by rfay** [reply, marked as answer]
There is no "native docker container". I assume you're talking about the docker server you have installed. If you show `docker version` I'll know more, and if you say what Linux distro you're using. Linux Docker install instructions are at https://ddev.readthedocs.io/en/stable/users/install/docker-installation/#docker-installation-linux

**Reply by bbholty1** [reply]
What I meant by native docker container is that it was compiled specifically for my Linux OS / Architecture / Distribution / Etc... to Play nicely and be compatible with every other piece of hardware and software on the system...

Docker version 24.0.7, build 24.0.7-0ubuntu4.1

I haven't retried using ddev yet at this point...  To Recreate the issue 
I know it previously complained and failed about some sort of container error specifically when I tried to config drupal.
ddev config --project-type drupal --docroot ...

I plan to reinstall ddev again and see if I still see the same  container complaint /  failure during the config and keep you posted.

Thanks Again For Helping Me Out!

**Reply by rfay** [reply]
The docker version you're using appears to be plain-vanilla docker-ce for Ubuntu, I wouldn't expect you to have problems.

If `docker ps` works, then DDEV will probably work.

**Reply by bbholty1** [reply]
I still got the same non-intuitive error message as the last time I tried.   :-(
> ddev config --project-type drupal --docroot ./                                                                                                                                                                           |~                                                                                                                                                                                                                             |~
Could not connect to a Docker provider. Please start or install a Docker provider.                                                                                                                                            |~                                                                                                                                                                                                                             |~
For install help go to: https://ddev.readthedocs.io/en/stable/users/install/docker-installation/ 


But then I remembered I hadn't added myself to the docker group yet on this PC... And after completing that step Iv'e now got ddev working fine! :-)

Thanks Again For You Help
-Bryan
